scripts/validate_test_schemas.sh

```diff
@@ -36,30 +39,37 @@ fi
 echo "--- Testing Schemas in $file_path ---"
 failed=0
 passed=0
+exit=0
```

Comment:
Can probably remove `exit=0` here as it's declared & initialised a little further up?